daisy_workflows/image_import/windows/translate.ps1

```diff
@@ -93,6 +93,11 @@ function Setup-NTP {
   Set-ItemProperty -Path $tzi_path -Name RealTimeIsUniversal -Value 1
 
   # Set up time sync...
+  # Ensure the time service isn't in a pending state; wait until it is not in a pending state.
+  while((Get-Service -Name "W32time").Status -match 'pending') {
+    Write-Host "W32time service is in pending state, waiting 1 second."
+    Start-Sleep -s 1
+  }
```

Comment:
Looks like these two pieces of code are the same. If so, extract them in a separate function and re-use.

Reply:
Absolutely, I'll make this into a utility function.

The service controller has global timeouts, so we shouldn't need to. By default it gives a service 20000ms to complete a state change, though that can be customized in the OS's settings.